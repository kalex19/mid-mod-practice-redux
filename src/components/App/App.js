import React from 'react';
import './App.css';
import Form from '../../containers/Form/Form';
import Container from '../Container/Container';

function App(){
	return (
		<div className="App">
			<Form />
			<Container />
		</div>
	);
}

export default App;
