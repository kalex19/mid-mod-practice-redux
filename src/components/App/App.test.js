import React from 'react';
import App from './App';
import { shallow } from 'enzyme';

describe('App', () => {
	let wrapper;

	beforeEach(() => {
		wrapper.shallow(<App />);
	});
	it('should match snapShot', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
