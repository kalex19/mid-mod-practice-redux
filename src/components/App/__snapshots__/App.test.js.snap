// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapShot 1`] = `undefined`;
